ui/scss/core/components/individual_sim_ui/_apl_rotation_picker.scss

```diff
@@ -57,6 +57,30 @@
 		width: unset;
 		margin: 0;
 	}
+
+	.apl-validation-information {
+		color: var(--bs-off-white) !important;
+		text-shadow:
+		0 0 var(--spacer-2) var(--bs-link-color),
+		0 0 var(--spacer-2) var(--bs-link-color),
+		0 0 var(--spacer-2) var(--bs-link-color);
```

Comment:
Why are there 3 shadows stacked on top of eachother? :O

Reply:
:sweat_smile: I copied them from here without reading the docs https://github.com/wowsims/cata/blob/910e5ce1caee314690b24ae64547b2ae7aa9dcf2/ui/scss/shared/_global_old.scss#L91-L97

Reply:
Changing to 1 shadow (left) looks better for Info but does make the shadow on Warning pretty faint compared to live (right):
![image](https://github.com/user-attachments/assets/7ff95f4f-2983-4669-a249-9129411dcd18)

Reply:
I didn't end up using Error in any current validations, but here's what it looks like single versus triple (also open to feedback on color scheme lol)

![image](https://github.com/user-attachments/assets/24fb17e3-0e66-494c-beef-7deefe8a286f)
![image](https://github.com/user-attachments/assets/fd22d13d-3f63-4266-ae80-532f4a39efe4)